java/java.lsp.server/nbproject/org-netbeans-modules-java-lsp-server.sig

```diff
@@ -23,6 +23,35 @@ meth public void notifyLater(org.openide.NotifyDescriptor)
 supr org.openide.DialogDisplayer
 hfds context
 
+CLSS public org.netbeans.modules.java.lsp.server.ui.AbstractGlobalActionContext
+cons public init()
+intf org.openide.util.ContextGlobalProvider
+intf org.openide.util.Lookup$Provider
+meth public org.openide.util.Lookup createGlobalContext()
+meth public org.openide.util.Lookup getLookup()
+meth public static <%0 extends java.lang.Object> {%%0} withActionContext(org.openide.util.Lookup,java.util.concurrent.Callable<{%%0}>)
+supr java.lang.Object
+hcls ContextHolder
+
+CLSS public org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer
+cons protected init()
+innr protected final View
+intf org.netbeans.spi.htmlui.HTMLViewerSpi<org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer$View,java.lang.Object>
+meth public <%0 extends java.lang.Object> {%%0} component(org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer$View,java.lang.Class<{%%0}>)
+meth public java.lang.Object createButton(org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer$View,java.lang.String)
+meth public java.lang.String getId(org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer$View,java.lang.Object)
+meth public org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer$View newView(org.netbeans.spi.htmlui.HTMLViewerSpi$Context)
+meth public void runLater(org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer$View,java.lang.Runnable)
+meth public void setEnabled(org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer$View,java.lang.Object,boolean)
+meth public void setText(org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer$View,java.lang.Object,java.lang.String)
+supr java.lang.Object
+hfds initial
+
+CLSS protected final org.netbeans.modules.java.lsp.server.ui.AbstractLspHtmlViewer$View
```

Comment:
sigtest doesn't like private `View` - making it `protected` to fix `ant gen-sigtests-release` problem introduced in 9e27e191c829978a9dbdcdaeee774d4208412348